gravitee-cockpit-connectors-ws/src/main/java/io/gravitee/cockpit/connectors/ws/WebSocketCockpitConnector.java

```diff
@@ -45,8 +46,9 @@
  * @author Jeoffrey HAEYAERT (jeoffrey.haeyaert at graviteesource.com)
  * @author GraviteeSource Team
  */
+// This class is instanciated as a Spring Component by Gravitee Node
+@SuppressWarnings("SpringJavaInjectionPointsAutowiringInspection")
```

Comment:
This SuppressWarning is exclusive to Intellij project configuration. I wouldn't add it.